README.md

```
@@ -118,6 +118,36 @@ nix run .#update
 - OS本体やカーネルは pacman（Linux）や macOS 標準管理に任せる
 - Home Manager によるリンクや設定は既存の dotfiles を上書きする場合があります
 
+## 運用ルール（品質ゲート）
+
+- PR では **Nix Flake Check**（`nix flake check`）と **treefmt check**（`nix fmt -- --ci`）を必須とします。
+- ローカルでも PR 前に次を実行してください。
+
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)